iron-menu-behavior.html

```diff
@@ -41,14 +41,21 @@
        */
       attrForItemTitle: {
         type: String
-      }
+      },
+
+      disabled: {
+        type: Boolean,
+        value: false,
+        observer: '_disabledChanged',
+      },
+
+      _previousTabIndex: Number,
```

Comment:
maybe remove from `properties` (and just put on prototype) until you need to observe it?